server/http.go

```diff
@@ -235,6 +269,54 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 	p.handleMeetingEnded(w, r, &meetingWebhook)
 }
 
+func (p *Plugin) handleParticipantJoinedLeftEvent(w http.ResponseWriter, event *zoom.ParticipantJoinedLeftEvent) {
+	participant := event.Payload.Object.Participant
+	userID, uerr := p.getMattermostUserIdFromZoomParticipant(participant)
+	if uerr != nil {
+		errMsg := "failed to get Mattermost user id for Zoom user"
+		p.API.LogWarn(errMsg, "err", uerr.Error())
+		http.Error(w, errMsg, http.StatusBadRequest)
```

Comment:
This may happen if there are many users in Zoom that are not part of Mattermost. I wonder if we should put this down to debug logs. Also, not sure how will interpret Zoom so many "bad request" error returns to the webhook.